app/src/main/res/values/strings.xml

```diff
@@ -143,34 +144,34 @@
+    <string name="detail_item_cannot_download">Cannot open or download attachment. The link expired and no local file could be found.</string>
     <string name="detail_item_cannot_open">Cannot open attachment: %1$s</string>
-    <string name="detail_item_cannot_open_not_found">Cannot open attachment: File may have been deleted, or there is no app to open the file.</string>
-    <string name="detail_item_cannot_open_click_url">Cannot open click URL: %1$s</string>
+    <string name="detail_item_cannot_open_not_found">Cannot open attachment: The file may have been deleted, or no installed app can open the file.</string>
+    <string name="detail_item_cannot_open_click_url">Cannot open clickable URL: %1$s</string>
     <string name="detail_item_cannot_save">Cannot save attachment: %1$s</string>
     <string name="detail_item_cannot_delete">Cannot delete attachment: %1$s</string>
-    <string name="detail_item_download_failed">Attachment download failed: %1$s</string>
-    <string name="detail_item_download_info_not_downloaded">not downloaded</string>
-    <string name="detail_item_download_info_not_downloaded_expired">not downloaded, link expired</string>
-    <string name="detail_item_download_info_not_downloaded_expires_x">not downloaded, expires %1$s</string>
+    <string name="detail_item_download_failed">Could not download attachment: %1$s</string>
+    <string name="detail_item_download_info_not_downloaded">Not downloaded.</string>
+    <string name="detail_item_download_info_not_downloaded_expired">Not downloaded. The link expired.</string>
+    <string name="detail_item_download_info_not_downloaded_expires_x">Not downloaded. Expires %1$s.</string>
     <string name="detail_item_download_info_downloading_x_percent">%1$d%% downloaded</string>
-    <string name="detail_item_download_info_deleted">deleted</string>
-    <string name="detail_item_download_info_deleted_expired">deleted, link expired</string>
-    <string name="detail_item_download_info_deleted_expires_x">deleted, link expires %1$s</string>
-    <string name="detail_item_download_info_download_failed">download failed</string>
-    <string name="detail_item_download_info_download_failed_expired">download failed, link expired</string>
-    <string name="detail_item_download_info_download_failed_expires_x">download failed, link expires %1$s</string>
+    <string name="detail_item_download_info_deleted">Deleted</string>
+    <string name="detail_item_download_info_deleted_expired">Deleted. The link expired.</string>
+    <string name="detail_item_download_info_deleted_expires_x">Deleted. The link expires %1$s.</string>
+    <string name="detail_item_download_info_download_failed">Download failed.</string>
+    <string name="detail_item_download_info_download_failed_expired">Download failed. The link expired.</string>
+    <string name="detail_item_download_info_download_failed_expires_x">Download failed. The link expires %1$s.</string>
```

Comment:
You have to revert all of these. These strings are concatenated to the attachment info box, separated by commas.
This is also just your personal preference :-/